docs/source/getting-started/examples_intro.rst

```diff
@@ -0,0 +1,85 @@
+Getting Started With Examples
+==========
+
+Motivation 
+----------
+
+In the following examples, we will use real data to demonstrate the utility of OpenDP functions. We will use the Labour Force Survey microdata released by Eurostat for a few reasons: 
+
+1. **Generality:** The dataset is relatively general, with variables and contexts accessible to users across various domains.
+2. **Sample Utility:** The public micro-data is a sample of the more thorough research microdata available through scientific use files. Methods developed with the public microdata are designed to also work on the scientific files. Researchers can request access to the full dataset through Eurostat. 
+
+Dataset Description 
+-------------------
+
+The data is organized by year and quarter for each nation in the European Union. For this tutorial, we sampled 200,000 individuals from the public microdata of France across all study years. 
+
+The data is protected using traditional statistical disclosure control methods such as global recoding, local suppression, and addition of noise. 
+
+Learn more about the dataset `here <https://ec.europa.eu/eurostat/web/microdata/public-microdata/labour-force-survey>`_! The `user guide <https://ec.europa.eu/eurostat/documents/1978984/6037342/EULFS-Database-UserGuide.pdf>`_ may also be helpful to learn more about the different variables.
+
+Core Variables 
+--------------
+There are many variables available. The examples will focus on the following key variables: 
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Variable
+     - Definition
+     - Coding
+   * - `SEX`
+     - Sex
+     - 1: Male \n 2: Female
+   * - `AGE`
+     - Age of the Individual During the Reference Week
+     - Single Years
+   * - `ILOSTAT`
+     - Labour Status During the Reference Week
+     - 1: Did any work for pay or profit during the reference week - one hour or more (including family workers but excluding conscripts on compulsory military or community service) \n 2: Was not working but had a job or business from which he/she was absent during the reference week (including family workers but excluding conscripts on compulsory military or community service) \n 3: Was not working because of lay-off \n 4: Was a conscript on compulsory military or community service \n 5: Other (15 years or more) who neither worked nor had a job or business during the reference week \n 9: Not applicable (child less than 15 years old)
+   * - `HWUSUAL`
+     - Number of Hours Per Week Usually Worked
+     - 00: Usual hours cannot be given because hours worked vary considerably from week to week or from month to month \n 01-98: Number of hours usually worked in the main job \n 99: Not applicable (WSTATOR=3-5,9) \n blank: No answer
+   * - `QUARTER`
+     - Fixed Reference Quarter
+     - Single Quarter
+   * - `YEAR`
+     - Fixed Reference Year
```

Comment:
- `\n` isn't special for sphinx. Replacing with semicolons.
- double backticks for code formatting here.
- Leave off `WSTATOR=3-5,9` if we don't define it.

```suggestion
   * - ``SEX``
     - Sex
     - 1: Male; 2: Female
   * - ``AGE``
     - Age of the Individual During the Reference Week
     - Single Years
   * - ``ILOSTAT``
     - Labour Status During the Reference Week
     - 1: Did any work for pay or profit during the reference week - one hour or more (including family workers but excluding conscripts on compulsory military or community service); 2: Was not working but had a job or business from which he/she was absent during the reference week (including family workers but excluding conscripts on compulsory military or community service); 3: Was not working because of lay-off; 4: Was a conscript on compulsory military or community service; 5: Other (15 years or more) who neither worked nor had a job or business during the reference week; 9: Not applicable (child less than 15 years old)
   * - ``HWUSUAL``
     - Number of Hours Per Week Usually Worked
     - 00: Usual hours cannot be given because hours worked vary considerably from week to week or from month to month; 01-98: Number of hours usually worked in the main job; 99: Not applicable; blank: No answer
   * - ``QUARTER``
     - Fixed Reference Quarter
     - Single Quarter
   * - ``YEAR``
     - Fixed Reference Year
```